drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw_B/host_rpu_sys_if.h

```diff
@@ -135,7 +135,7 @@ enum rpu_op_mode {
  *
  * To obtain statistics relevant to the operation mode set via op_mode
  * parameter.
- * Statistics will be updated in: /sys/kernel/debug/img/wlan/stats
+ * Statistics will be updated in: /sys/kernel/debug/nrf_wifi/wlan/stats
  */
```

Comment:
This should be removed since this is specific to linux?

Reply:
Done